src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -22,9 +22,8 @@ import { ONE_HUNDRED_PERCENT, ZERO_PERCENT } from 'constants/misc'
 import { PERCENTAGE_PRECISION } from 'constants/index'
 
 enum ErrorMsgs {
-  Initial = 'Insufficient balance to cover the selected investment amount. Either modify the investment amount or unselect the investment option to move forward',
```

Comment:
We might want to tells users that they can either modify OR unselect it, which can only be done in the first screen.

Also, you can't change the amount when claiming for someone else, we should distinguish that

Reply:
> We might want to tells users that they can either modify OR unselect it, which can only be done in the first screen.
> Also, you can't change the amount when claiming for someone else, we should distinguish that

My thinking is that messages should be shorter and `Insufficient balance to cover investment amount` works for 100% investments where you don't have enough tokens, or for self claiming. 

I'll ask for suggestions, not too strongly opinionated so happy to go either way as long as we keep it short.

Reply:
short is fine by me